fixtures/attribute-behavior/AttributeTableSnapshot.md

```diff
@@ -5427,20 +5427,20 @@
 | Test Case | Flags | Result |
 | --- | --- | --- |
 | `inert=(string)`| (changed)| `<boolean: true>` |
-| `inert=(empty string)`| (changed)| `<boolean: true>` |
+| `inert=(empty string)`| (initial)| `<boolean: false>` |
```

Comment:
seems like this doesn't warn but I it seems it should?

Reply:
We generally don't warn for empty strings on boolean attributes (e.g. check the entry for `hidden` or `open`). There's probably a case to make an exception here because of how long we had the workaround in place. But then how long do we keep the warning in? 

If you previously got type-checking to work via module augmentation, you'd get type errors. But if you always had the `@ts-expect-error` in place, nothing will change.

Reply:
Probably just one major. The problem is libraries trying to support multiple versions of React. We don't like them to have to version check React but they kind of need to here b/c the behavior of the workaround will result in the opposite behavior in whatever version ships this. We can expect that after 1 major folks will either have bumped their minimum supported version or adopted some kind of version check

Reply:
Added a dedicated warning for new boolean props:

> Received an empty string for a boolean attribute `inert`. This will treat the attribute as if it were false. Either pass `false` to silence this warning, or pass `true` if you used an empty string in earlier versions of React to indicate this attribute is true.